### DISCUSSION TITLE
Neck Mechanism

### DISCUSSION THREAD

**Discussion by Chompalot** [makerforgetech/modular-biped#50] [Ideas]
I was trying to see how your mechansim worked, but ended up modelling a rough mockup of an alternate design
![Concept-pic1](https://github.com/dmt-labs/modular-biped/assets/34891324/fe785a77-a89b-49ef-bd63-f9bdf0745d81)
![Concept-pic2](https://github.com/dmt-labs/modular-biped/assets/34891324/7699a66c-1406-4ca2-9991-c1c0946f1f01)

This discussion wouldn't allow me to upload the stl assembly, so I took a couple of snap shots

**Comment by WhoIsMrSentry** [comment]
Thanks for being in the community and contributing. @Chompalot very good congratulations, do you have a chance to send me the sample files, maybe I will change the neck mechanism, what I did was not very good 

I want to play on it and make it better, you can upload it to the cloud like drive and link it, this platform does not allow stl format as you stated

**Comment by Chompalot** [comment]
https://www.dropbox.com/scl/fi/vdp4f34a1gtvv625lbafo/assy-neck.stl?rlkey=506ok51v11vla3r0sei0ftts6&dl=0

here's a link to the stl assembly, hopefully you'll be able to access them.
Please note it's only roughly modelled to show ideas, for discussion.

I really need the stl's for the head and body etc, I've only found the legs so far. So I can make an assembly to see what it looks like together.

**Reply by WhoIsMrSentry** [reply]
There are models in my repo, but there are Turkish articles, you can see which model is what by using translation.
[3D-Prints](https://github.com/SentryCoderDev/SentryBOT/tree/SentryBOT-V3/3D%20baskılar)

**Comment by danic85** [comment]
Thanks for sharing this, it looks nice! Just be careful with weight for modifications as every gram increases strain on the servos and shifts the center of gravity further when the head moves around which can lead to imbalance. 

> I really need the stl's for the head and body

These are all available in links on the wiki attached to this repo. Let me know if you cant find anything.

**Reply by Chompalot** [reply]
Thanks Dan, I was poking around this afternoon and found the Wiki with the links to the stl's.
I also found the Coral accelerator I bought for K9, but never got around to. Just waiting for the PLA+ to turn up now.

**Comment by Chompalot** [comment]
Hi Dan the Hips and Body links seem to have bits missing

**Reply by WhoIsMrSentry** [reply]
If you want, you can look at my repo, I remember there were some missing parts, so I made it myself and added it. it's there but don't take the head part because I made the head for jetson nano
[3D-prints](https://github.com/SentryCoderDev/SentryBOT/tree/SentryBOT-V3/3D%20baskılar)

**Reply by danic85** [reply]
@Chompalot what is missing? Those are the files I used to print so I'm surprised about that.

**Reply by Chompalot** [reply]
In the body there's just the Drive (full).stl   so none of the body bits.
In the hips there are Bearing Spacer, Drive (full), Lower Drive and Screw Spacer. the bit to join the 2 legs is missing.

Thanks Sentry, I may have a look at your files but it'd be good to get the parts linked for others to find.

**Comment by danic85** [comment]
Ah yes that link is not correct. Try this:

https://cad.onshape.com/documents/2847233f33aab45fe405538e/w/bee9409e13e5a847e8629e19/e/4955af09cbb743f37936c9fa


________________________________

**Reply by Chompalot** [reply]
Thats got it, thanks
